src/mlpack/core/data/split_data.hpp

```diff
@@ -61,9 +63,13 @@ void Split(const arma::Mat<T>& input,
   trainLabel.set_size(trainSize);
   testLabel.set_size(testSize);
 
-  const arma::Col<size_t> order =
-      arma::shuffle(arma::linspace<arma::Col<size_t>>(0, input.n_cols - 1,
-                                                      input.n_cols));
+  arma::Col<size_t> order;
+  if (shuffleData)
+    order = arma::shuffle(arma::linspace<arma::Col<size_t>>(0,
+        input.n_cols - 1, input.n_cols));
+  else
+    order = arma::linspace<arma::Col<size_t>>(0, input.n_cols - 1,
+        input.n_cols);
```

Comment:
```suggestion
  if (shuffleData)
  {
    order = arma::shuffle(arma::linspace<arma::Col<size_t>>(0,
        input.n_cols - 1, input.n_cols));
  }
  else
  {
    order = arma::linspace<arma::Col<size_t>>(0, input.n_cols - 1,
        input.n_cols);
  }  
```

I think that is easier to read.

Reply:
Done